### ISSUE TITLE
IL2CPP Managed code stripping throws errors in build runtime

### ISSUE THREAD

**Issue by zarrashi** [ValveSoftware/steamvr_unity_plugin#396]
When using IL2CPP to build and a managed code stripping level of anything but low will cause generated action files to be stripped. This can be fixed by adding a link.xml file to the SteamVR folder such as:

```XML
<linker>
       <assembly fullname="SteamVR" preserve="all"/>
       <assembly fullname="SteamVR_Actions" preserve="all"/>
</linker>
```

**Comment by zarrashi** [comment]
note that this workaround only works if the assembly definition files for SteamVR and SteamVR_Actions aren't renamed or deleted.

**Comment by zite** [comment]
Thanks, I'll get this added to the next release.

**Comment by zite** [comment]
Fixed in the recent 2.3.2 asset store release https://github.com/ValveSoftware/steamvr_unity_plugin/releases/tag/2.3.2
